tcms/testplans/views.py

```diff
@@ -95,6 +96,7 @@ def get_context_data(self, **kwargs):
         context_data = {
             'form': form,
             'plan_types': PlanType.objects.all().only('pk', 'name').order_by('name'),
+            'defaultPageSize': settings.DEFAULT_PAGE_SIZE
```

Comment:
With this approach you will have to modify context for every page which needs access to this variable. Instead there is a context processor which exports all settings to templates. See `tcms/core/context_processors.py` and conversation on Slack from Feb 13th.